app/src/main/java/de/xikolo/services/DownloadService.java

```diff
@@ -69,6 +72,16 @@ public void onCreate() {
 
         // Don't use HttpLoggingInterceptor, crashes with OutOfMemoryException!
         OkHttpClient client = new OkHttpClient.Builder()
+                .addInterceptor(chain -> {
+                    Request original = chain.request();
+
+                    Request.Builder builder = original.newBuilder();
+                    if (original.url().host().equals(getString(R.string.app_host)) && UserManager.isAuthorized()) {
+                        builder.header(Config.HEADER_AUTH, Config.HEADER_AUTH_VALUE_PREFIX_JSON_API + UserManager.getToken());
+                    }
+
+                    return chain.proceed(builder.build());
+                })
```

Comment:
This logic is kinda duplicated like what we do in our `ApiService`. I'm wondering if we could move this interceptor into an object in `ApiService` which can be used here and there. Everything else what happens in `ApiService` can be moved into another interceptor there.

Reply:
Created an static `authenticationInterceptor` in `ApiService` which attaches the `AUTH` header to every request to an own server.